packages/allure-cucumberjs/src/CucumberJSAllureReporter.ts

```diff
@@ -246,7 +246,7 @@ export class CucumberJSAllureFormatter extends Formatter {
     }
 
     if (labels.length > 0) {
-      labels.forEach((label) => this.currentTest?.addLabel(label.name, label.value));
+      labels.forEach((label) => currentTest?.addLabel(label.name, label.value));
```

Comment:
currentTest wasn't in the right context, which was causing a unit test to fail in the allure-cucumberjs unit test suite.